src/main/java/org/zalando/nakadi/domain/SubscriptionAuthorization.java

```diff
@@ -43,6 +48,14 @@ public SubscriptionAuthorization(@JsonProperty("admins") final List<Authorizatio
         return readers;
     }
 
+    @JsonIgnore
+    public List<AuthorizationAttribute> getAll() {
+        return Stream.of(
+                Optional.ofNullable(readers).orElse(new ArrayList<>()),
+                Optional.ofNullable(admins).orElse(new ArrayList<>()))
```

Comment:
I think it doesn't make sense to check for null something that is not marked as `@Nullable`
In Nakadi we keep the not-nullable-by-default practice which allows to keep code clean from unnecessary null checks.

Reply:
You're right. Fixed in `de2a148`